tests/settings/test_settings_embedders.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from meilisearch.models.index import Embedders
+
+
+@pytest.mark.usefixtures("enable_vector_search")
+def test_get_default_embedders(empty_index):
+    """Tests getting default embedders."""
+    response = empty_index().get_embedders()
+
+    assert response is None
+
+
+@pytest.mark.usefixtures("enable_vector_search")
+def test_update_embedders_with_user_provided_source(new_embedders, empty_index):
+    """Tests updating embedders."""
+    index = empty_index()
+    response_update = index.update_embedders(new_embedders)
+    update = index.wait_for_task(response_update.task_uid)
+    response_get = index.get_embedders()
+    assert update.status == "succeeded"
+    assert response_get == Embedders(embedders=new_embedders)
+
+
+@pytest.mark.usefixtures("enable_vector_search")
+@pytest.mark.usefixtures("new_embedders")
```

Comment:
This `usefixtures` isn't needed for `new_embedders` since it is included in the arguments.

`usefixtures` is for when you need to use a fixture, but never reference it the body of the function.